lib/client/ClientApp.js

```diff
@@ -42,6 +42,10 @@ var ClientApp = App.extend({
             root: appRoot || "",
             pushState: this.isPushStateAvailable()
         });
+
+        window.Brisket = {
+            version: require("../brisket").version
```

Comment:
I thought about that, too. Apparently, as a result of the test, CommonJS is smart enough to not require an infinite loop.